data/blog/electron-12-0.md

```diff
@@ -0,0 +1,115 @@
+* `extensions` API changes:
+    * Added support for some `chrome.management` APIs. [#25098](https://github.com/electron/electron/pull/25098)
+* `MenuItem` API changes:
+    * Added support for showing macOS share menu. [#25629](https://github.com/electron/electron/pull/25629) 
+* `net` API changes:
+    * Added a new `credentials` option for `net.request()`. [#25284](https://github.com/electron/electron/pull/25284) 
+    * Added `net.online` for detecting whether there is currently internet connection. [#21004](https://github.com/electron/electron/pull/21004) 
+* `power-monitor` API changes:
+    * Added `powerMonitor.onBatteryPower`. [#26494](https://github.com/electron/electron/pull/26494) 
+    * Added fast user switching event to powerMonitor on macOS. [#25321](https://github.com/electron/electron/pull/25321)
+* `session` API changes:
+    * Added `allowFileAccess` option to `ses.loadExtension()` API. [#27702](https://github.com/electron/electron/pull/27702)
+    * Added `display-capture` API for `session.setPermissionRequestHandler`. [#27696](https://github.com/electron/electron/pull/27696) 
+    * Added a `disabledCipherSuites` option to `session.setSSLConfig`. [#25818](https://github.com/electron/electron/pull/25818) 
+    * Added `extension-loaded`, `extension-unloaded`, and `extension-ready` events to `session`. [#25385](https://github.com/electron/electron/pull/25385) 
+    * Added `session.setSSLConfig()` to allow configuring SSL. [#25461](https://github.com/electron/electron/pull/25461)
+    * Added support for explicitly specifying `direct`, `auto_detect` or `system` modes in `session.setProxy()`. [#24937](https://github.com/electron/electron/pull/24937)
+    * Added Serial API support. [#25237](https://github.com/electron/electron/pull/25237) 
+    * Added APIs to enable/disable spell checker. [#26276](https://github.com/electron/electron/pull/26276) 
+* `shell` API changes:
+    * Added a new asynchronous `shell.trashItem()` API, replacing the synchronous `shell.moveItemToTrash()`. [#25114](https://github.com/electron/electron/pull/25114) 
+* `webContents` API changes:
+    * Added a small console hint to console to help debug renderer crashes. [#25317](https://github.com/electron/electron/pull/25317)
+    * Added `frame` and `webContents` properties to the details object in webRequest handlers. [#27334](https://github.com/electron/electron/pull/27334) 
+    * Added `webContents.forcefullyCrashRenderer()` to forcefully terminate a renderer process to assist with recovering a hung renderer. [#25580](https://github.com/electron/electron/pull/25580)
+    * Added `setWindowOpenHandler` API for renderer-created child windows, and deprecate `new-window` event. [#24517](https://github.com/electron/electron/pull/24517) 
+* `webFrame` API changes:
+    * Added spellcheck API to renderer. [#25060](https://github.com/electron/electron/pull/25060) 
+* `webFrameMain` API changes:
```

Comment:
Since `webFrameMain` is a completely new API, it might make sense to restructure this section to present it as a whole rather than each individual PR.

We spend a lot of lines here listing additions that were added in subsequent PRs, but that information doesn't mean much for users of the last Electron stable, since everything is new to them.

Reply:
Great point; I'll restructure this into an explanatory paragraph and will add it to the top of the API changes section.